tests/integration/pusher/trigger.js

```diff
@@ -19,7 +19,7 @@ describe("Pusher", function () {
   })
 
   describe("#trigger", function () {
-    it("should send the event when no callback is given", function (done) {
+    it("should send the event when promise isn't handled", function (done) {
```

Comment:
Is this test still necessary? Seems a bit pointless

Reply:
Agreed, I'll remove it